packages/wp-dashboard/src/api/authors.js

```diff
@@ -27,7 +27,7 @@ export function getAuthors(config, search) {
   return apiFetch({
     path: addQueryArgs(config.api.users, {
       per_page: '100',
-      who: 'authors',
+      capabilities: config.capabilityName,
```

Comment:
Don't forget to actually pass this to the config in `Dashboard`

Same naming concern otherwise